Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "euc"
-version = "0.5.0"
+version = "0.5.1"
```

Comment:
You can't do a non-major release for a publicly exposed dependency that got a major release.

Reply:
didn't it only change a minor version ? from 0.10 to 0.11
not sure if this could lead to a patch version though

Reply:
Cargo considers the first non-zero version to be the "major version" (in terms of compatibility). So 0.10 to 0.11 is a breaking change release. 0.5.0 to 0.5.1 isn't. Since the types of vek are exposed publicly in euc, this is going to break code that depends on euc (like my crate).

Reply:
I'll make this a major version change for the next release.